show/main.py

```diff
@@ -393,6 +393,49 @@ def run_command_in_alias_mode(command):
         sys.exit(rc)
 
 
+def get_bgp_summary_extended(command):
+    """
+    Adds Neighbor name to the show ip[v6] bgp summary command
+    :param command: command to get bgp summary
+    """
+    try:
+        p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Also can we modifiy run_command to take a bool arg and return the output but not echo if True, then you can reuse the it at many places.:


def run_command(command, display_cmd=False, return_cmd=False):
        <snip>
        if output and return_cmd:
            return output
        elif output:
            click.echo(output.rstrip('\n'))

Reply:
good one. modifying in next iteration